Dockerfile.openvino

```diff
@@ -185,3 +202,86 @@ RUN curl https://storage.googleapis.com/mediapipe-assets/ssdlite_object_detectio
 
 # BUILD python framework
 RUN python setup.py install
+## End of build image ################################################################
+######################################################################################
+## Start of demos image - minimal size ###############################################
+FROM $BASE_IMAGE as demos
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install --no-install-recommends -y curl libpugixml1v5 libtbb2 libxml2-dev uuid-dev uuid libssl-dev --no-install-recommends
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    libopencv-core-dev \
+    libopencv-highgui-dev \
+    libopencv-calib3d-dev \
+    libopencv-features2d-dev \
+    libopencv-imgproc-dev \
+    libopencv-video-dev \
+    ffmpeg
+
+# Python samples requirements start
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    python3 \
+    python3-opencv \
+    python3-pip
+
+RUN pip3 install attrs matplotlib protobuf==3.20.2 numpy==1.24.4
+RUN ln -s /usr/bin/python3 /usr/bin/python
+# Python samples requirements end
+
+# Openvino
+RUN apt-get update && apt-get install -y libavformat-dev libdc1394-dev libgtk2.0-dev
+RUN mkdir -p /opt/intel/openvino/runtime/lib/intel64/
+COPY --from=build /opt/intel/openvino/runtime/lib/intel64/ /opt/intel/openvino/runtime/lib/intel64/
+RUN mkdir -p /opt/intel/openvino/runtime/3rdparty/tbb/lib/
+COPY --from=build /opt/intel/openvino/runtime/3rdparty/tbb/lib/ /opt/intel/openvino/runtime/3rdparty/tbb/lib/
+
+# 2GB with python 3.8 - currently excluded as only .so libs are copied
+COPY --from=build /usr/local/lib/*.so* /usr/local/lib/
```

Comment:
what is included here?

Reply:
358  MB
150 MB is python3.8

libazurestorage.so      libopencv_calib3d.so.4.7.0  libopencv_features2d.so.4.7.0  libopencv_highgui.so.4.7.0    libopencv_imgproc.so.4.7.0  libopencv_video.so.4.7.0    libopencv_ximgproc.so.4.7.0
libazurestorage.so.7    libopencv_calib3d.so.407    libopencv_features2d.so.407    libopencv_highgui.so.407      libopencv_imgproc.so.407    libopencv_video.so.407      libopencv_ximgproc.so.407
libazurestorage.so.7.5  libopencv_core.so           libopencv_flann.so             libopencv_imgcodecs.so        libopencv_optflow.so        libopencv_videoio.so        python3.8
libcpprest.so           libopencv_core.so.4.7.0     libopencv_flann.so.4.7.0       libopencv_imgcodecs.so.4.7.0  libopencv_optflow.so.4.7.0  libopencv_videoio.so.4.7.0
libcpprest.so.2.10      libopencv_core.so.407       libopencv_flann.so.407         libopencv_imgcodecs.so.407    libopencv_optflow.so.407    libopencv_videoio.so.407
libopencv_calib3d.so    libopencv_features2d.so     libopencv_highgui.so           libopencv_imgproc.so          libopencv_video.so          libopencv_ximgproc.so